sdks/python/apache_beam/ml/transforms/handlers.py

```diff
@@ -83,20 +85,48 @@
 tft_process_handler_output_type = typing.Union[beam.Row, Dict[str, np.ndarray]]
 
 
+class DataCoder:
+  def __init__(
+      self,
+      exclude_columns,
+      coder=coders.registry.get_coder(Any),
+  ):
+    """
+    Uses PickleCoder to encode/decode the dictonaries.
```

Comment:
```suggestion
    Encodes/decodes items of a dictionary into a single element.
```

We use FastPrimitiveCoder. you could say something generic.